apps/test-viewer/src/components/Viewer.tsx

```diff
@@ -89,6 +90,34 @@ function onIModelConnected(imodel: IModelConnection) {
   setTimeout(() => {
     IModelConnection.onOpen.raiseEvent(imodel);
   }, 1000);
+  const setupFormatsProvider = async () => {
+    try {
+      const schema = await imodel.schemaContext.getSchema(new SchemaKey("AecUnits", SchemaMatchType.Latest));
+      if (schema) {
```

Comment:
What about:
```ts
if (!schema)
  throw new Error("No schema");
```
That will do two things:

1. Unindent the active code by one level
2. Do the console log if schema isn't defined

Note: since the error is being ignored in the catch handler, it doesn't need to be fancy.

Reply:
Oops; I see the error is no longer ignored in the catch handler; you can put a more descriptive error here.